src/data/alpacaLpPools.json

```diff
@@ -14,7 +14,7 @@
     "lp1": {
       "address": "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c",
       "oracle": "tokens",
-      "oracleId": "WBNB",
+      "oracleId": "BNB",
```

Comment:
1. These json files are semi-generated. 
2. Change like this adding manual work on plate of every vault developer which is hard to communicate and ensure. So data here inevitably will become inconsistent.

Reply:
The alternative I see is changing the keys once loaded

Reply:
But it requires a similar type of tuning, getting all the native/wrapped tokens and setting oracleIds to the same. It would mean the jsons themselves aren't useful as they are and data is dropped